webcompat/webhooks/__init__.py

```diff
@@ -16,13 +16,10 @@
 
 from webcompat.webhooks.helpers import get_issue_info
 from webcompat.webhooks.helpers import is_github_hook
-from webcompat.webhooks.helpers import new_opened_issue
+from webcompat.webhooks.helpers import process_issue_action
```

Comment:
thanks, `new_opened_issue` was not a great name.

Reply:
Thanks on commenting in advance to my future move which is for today aka change `new_opened_issue` to `tag_new_public_issue`;)